packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -307,19 +307,22 @@ function VM-Install-Shortcut{
         if ($runAsAdmin) {
             $shortcutArgs.RunAsAdmin = $true
         }
-
         Install-ChocolateyShortcut @shortcutArgs
 
     } else {
+        if (-not $executableDir) {
```

Comment:
what about moving this before the `if ($consoleApp -or $powershell)` instead of duplicating it?